var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -740,8 +744,9 @@ class Opencv(CMakePackage, CudaPackage):
     depends_on("cudnn@:7.3", when="@3.3.1:3.4+cudnn")
     depends_on("cudnn@:6", when="@:3.3.0+cudnn")
     depends_on("eigen", when="+eigen")
-    depends_on("ffmpeg+avresample", when="+ffmpeg")
-    depends_on("ffmpeg@:4+avresample", when="@:4.5+ffmpeg")
+    depends_on("ffmpeg", when="+ffmpeg")
+    depends_on("ffmpeg@:5", when="@:4.7+ffmpeg")
+    depends_on("ffmpeg@:4+avresample", when="@:4.6+ffmpeg")
```

Comment:
Do you have any source for ffmpeg support or is this scraped from the changelog? Just want to know where to check in the future

Reply:
Just trial and error. I didn't think about intelligent shortcuts such as consulting documentation.